snap/info.go

```diff
@@ -249,10 +249,25 @@ func (app *AppInfo) WrapperPath() string {
 	return filepath.Join(dirs.SnapBinariesDir, binName)
 }
 
+func (app *AppInfo) invocation(command string) string {
+	securityTag := app.SecurityTag()
+	return fmt.Sprintf("/usr/bin/ubuntu-core-launcher %s %s %s", securityTag, securityTag, filepath.Join(app.Snap.MountDir(), command))
+
+}
+
 // Invocation returns the launcher command line to use when invoking the app binary.
 func (app *AppInfo) Invocation() string {
-	securityTag := app.SecurityTag()
-	return fmt.Sprintf("ubuntu-core-launcher %s %s %s", securityTag, securityTag, filepath.Join(app.Snap.MountDir(), app.Command))
+	return app.invocation(app.Command)
+}
+
+// StopInvocation returns the launcher command line to use when invoking the app stop command binary.
+func (app *AppInfo) StopInvocation() string {
+	return app.invocation(app.Stop)
+}
+
+// PostStopInvocation returns the launcher command line to use when invoking the app post-stop command binary.
+func (app *AppInfo) PostStopInvocation() string {
```

Comment:
Can we use the following names:
* Command
* StopCommand
* PostStopCommand
* LauncherCommand
* LauncherStopCommand
* LauncherPostStopCommand

These seem to make their meaning and cross-distinction more evident.

Okay to be on a follow up.